src/__tests__/__components__/AppBars/__snapshots__/Navbar.test.js.snap

```diff
@@ -0,0 +1,214 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This file should be put in .gitignore

Reply:
Okay, by the way, can I ignore each `.js.snap` file or I only can ignore this?
I think there might be other snapshot files which were pushed.

Reply:
@Musigwa All of them should be ignored